.env.example

```diff
@@ -120,13 +120,13 @@ EXIF_EXIFTOOL_TIMEOUT=10
 
 ###> symfony/framework-bundle ###
 APP_ENV=prod
-APP_SECRET=!CHANGE_SECRET!
+APP_SECRET=!ChangeSecret!
 ###< symfony/framework-bundle ###
 
 ###> doctrine/doctrine-bundle ###
 # Format described at https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url
-POSTGRES_DB=kbin
-POSTGRES_USER=kbin
+POSTGRES_DB=mbin
+POSTGRES_USER=mbin
 POSTGRES_PASSWORD=!ChangeThisPostgresPass!
 # IMPORTANT: You MUST configure your PostgreSQL server version!
 POSTGRES_VERSION=16
```

Comment:
Latest version of postgres is v17. Maybe its a good idea to new people to just start using the latest stable PostgreSQL version. Which is again `17`.

Reply:
Actually, I did update it to `17` in the `.env.example_docker` file, but good catch, I will update it here as well for bare metal.